{"uuid": "e4ede3c7-7807-4c12-9257-39c65cfb235f", "children": ["14b517d9-4dac-41bb-9189-2025cbc6ef6d"], "befores": [{"name": "_xunit_setup_class_fixture_TestSwap", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/_pytest/fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/_pytest/python.py\", line 854, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/_pytest/python.py\", line 776, in _call_with_optional_argument\n    func(arg)\n  File \"/home/runner/work/dodo-test/dodo-test/Test/TestCase/test_swap.py\", line 40, in setup_class\n    WalletOperate().connectWallet(self.driver)\n  File \"/home/runner/work/dodo-test/dodo-test/Common/wallet_operate.py\", line 59, in connectWallet\n    Base.find_element('xpath,//button[text()=\"下一步\"]',driver).click()\n  File \"/home/runner/work/dodo-test/dodo-test/Base/base.py\", line 29, in find_element\n    element = WebDriverWait(driver, 30, 1).until(EC.presence_of_element_located((by, value)))\n  File \"/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/selenium/webdriver/support/wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1650038315864, "stop": 1650038368222}], "start": 1650038315864, "stop": 1650038368240}